skbio/tree/_gme.py

```diff
@@ -0,0 +1,452 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2013--, scikit-bio development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+import numpy as np
+
+from skbio.tree import TreeNode
+
+
+def gme(dm, allow_edge_estimation=True):
+    r"""Perform greedy minimum evolution (GME) on a phylogenetic tree.
```

Comment:
GME takes a distance matrix and produces a phylogenetic tree. Therefore "Perform GME on a phylogenetic tree." may not be accurate. How about "Perform GME for phylogenetic reconstruction".?